packages/core/src/contracts/tcr/listing.ts

```diff
@@ -46,76 +46,72 @@ export class Listing {
 
   //#region EventStreams
 
-  public applications(): Observable<TimestampedEvent<CivilTCR.LogEvents._Application>> {
-    return this.tcrInstance._ApplicationStream({ listingAddress: this.listingAddress }, { fromBlock: 0 }).map(e => {
+  public applications(fromBlock: number): Observable<TimestampedEvent<CivilTCR.LogEvents._Application>> {
```

Comment:
imo all of these should have a default parameter of 0 (or eventually... the block that we deploy the TCR contract at)